pydantic_evals/pydantic_evals/evaluators/llm_as_a_judge.py

```diff
@@ -100,6 +93,59 @@ async def judge_output(
 )
 
 
+def _build_prompt(
+    inputs: UserContent | Sequence[UserContent] | None,
```

Comment:
Can we make this `Any` again? `pydantic_evals` is not required to be used with `pydantic_ai`, the input could technically be anything, we just want to special-case the `pydantic_ai` values that we know how to format for the model